### ISSUE TITLE
Still no C++/WinRT?

### ISSUE THREAD

**Issue by clarkezone** [MicrosoftDocs/winrt-api#1625]
The lack of C++/WinRT content in the WinRT documentation is embarrassing.  Please add some.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: bae2249b-5542-c60a-056a-a19174834b24
* Version Independent ID: 112c2c8c-6076-a04c-3977-9f5efb8b387a
* Content: [ApplicationView.VisibleBoundsChanged Event (Windows.UI.ViewManagement) - Windows UWP applications](https://docs.microsoft.com/en-us/uwp/api/windows.ui.viewmanagement.applicationview.visibleboundschanged?view=winrt-18362)
* Content Source: [winrt-api-build/xml/Windows.UI.ViewManagement/ApplicationView.xml](https://github.com/MicrosoftDocs/winrt-api/blob/docs/winrt-api-build/xml/Windows.UI.ViewManagement/ApplicationView.xml)
* Product: **uwp**
* Technology: **winrt-reference**
* GitHub Login: @Karl-Bridge-Microsoft
* Microsoft Alias: **kbridge**

**Comment by stevewhims** [comment]
Hi, @clarkezone, and thanks for filing this issue. Let's chat internally about a plan, and then I can come back here later and update this thread.

-Steve

**Comment by stevewhims** [comment]
There were a few remaining C++/CX snippets that didn't *also* have a C++/WinRT version. Those last ones are now on their way through the publishing pipeline. Once those are done, I'll update this and close.

Thanks!
Steve

**Comment by clarkezone** [comment]
Thx.  Did you find the various missing ones for CoreWindow properties?

Sent from my iPad

On Apr 28, 2020, at 18:14, Steven White <notifications@github.com> wrote:

﻿

There were a few remaining C++/CX snippets that didn't also have a C++/WinRT version. Those last ones are now on their way through the publishing pipeline. Once those are done, I'll update this and close.

Thanks!
Steve

**Comment by stevewhims** [comment]
So, the plan is to go ahead and pursue internally any further improvements that we need. But this recent round of feedback has been great. It has highlighted an additional set of in-topic snippets that were either a) mis-marked as being plain C++ when they were actually already C++/WinRT, or b) were mis-marked as plain old C++ when they were actually C++/CX snippets that needed porting to C++/WinRT. Now that I've performed a rigorous search of the repo, and found and fixed all incorrect language tags, and ported all remaining snippets (except in 2 rare cases where the API is deprecated), I think we've made good progress.

So I'll go ahead and close this issue for now.

Thanks again!
Steve
